examples/ensemble/plot_gradient_boosting_regression_multioutput.py

```diff
@@ -0,0 +1,60 @@
+"""
+=========================================
+Multi-output Gradient Boosting Regression
+=========================================
+
+An example to illustrate multi-output regression with gradient boosting.
+
+This example illustrates the use of the
+:ref:`multioutput.MultiOutputRegressor <_multiclass>` meta-estimator
+to perform multi-output regression with an estimator that does not
+natively supoort it. For comparison a regression tree based model is
```

Comment:
![figure_1](https://cloud.githubusercontent.com/assets/1448859/12911127/590e0b26-cf10-11e5-889d-8be24af0b282.png)

This is RF vs MultiOutput+RF, all points are on top of each other. Does this make the plot more or less interesting?

Interesting observation, doing the same with a DT you don't get exactly the same result, which I think is related to how the tree's `random_state` is set in the two cases.